Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -60,6 +60,17 @@ public class ModelGroup {
             return model;
           }
         }
+        //Todo: if spawning enabled
+        for (int i = 0; i < modelsCheckedOut.Count; i++) {
+          if (modelsCheckedOut[i].Handedness == chirality && modelsCheckedOut[i].HandModelType == modelType) {
+            IHandModel modelToSpawn = modelsCheckedOut[i];
+            GameObject spawnedGO = GameObject.Instantiate(modelToSpawn.gameObject);
```

Comment:
You can actually just instantiate the hand model directly and it will automatically instantiate the game object it is on and return the new hand model.